plugin/cache/setup.go

```diff
@@ -176,6 +177,19 @@ func cacheParse(c *caddy.Controller) (*Cache, error) {
 					ca.percentage = num
 				}
 
+			case "serve_stale":
+				args := c.RemainingArgs()
+				if len(args) != 1 {
+					return nil, errors.New("syntax is: serve_stale <duration>")
+				}
+				d, err := time.ParseDuration(args[0])
+				if err != nil {
+					return nil, fmt.Errorf("invalid duration: %v", args[0])
```

Comment:
just return `err` as in the other parseDurations here.

Reply:
done